docs/content/release-process.md

```diff
@@ -9,41 +9,77 @@ url = "https://github.com/fermyon/spin/blob/main/docs/content/release-process.md
 To cut a release of Spin, you will need to do the following:
 
 1. Create a pull request that changes the version number for your new version
-   (e.g. `0.10.0-pre0` could become either `0.9.1` for a patch release or
-   `0.10.0` for a "major" release)
+   (e.g. `1.1.0-pre0` could become either `1.0.1` for a patch release or
+   `1.1.0` for a "major" release)
```

Comment:
why major instead of minor?

Reply:
Thanks for catching; updated to 'minor'.  Although I suppose we could also add the major release (eg 2.0.0) mention, I figured that will be a ways off and not a regular occurrence.